+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Wed Feb 15 01:44:03 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL javapackages-local
+ pkgs=javapackages-local
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs javapackages-local --node 1 --tempfile ''
Wed Feb 15 01:44:32 2023 - INFO  - pkgs:(javapackages-local) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ LOG_INFO 'Finish preparing the test environment.'
+ message='Finish preparing the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish preparing the test environment.'
Wed Feb 15 01:44:35 2023 - INFO  - Finish preparing the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Wed Feb 15 01:44:37 2023 - INFO  - Start to run test.
+ abs2rel /1/2/3/a/b/c /1/2/3
+ grep a/b/c
a/b/c
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ abs2rel foo/bar foo/baz
+ grep ../bar
../bar
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ gradle-local build

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'oe_test_javapackages-local'.
> java.io.FileNotFoundException: /usr/share/gradle/lib/plugins/gradle-diagnostics-4.4.1.jar (No such file or directory)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 41s
+ xmvn-builddep --help
+ grep 'Usage: /usr/bin/xmvn-builddep'
Usage: /usr/bin/xmvn-builddep [build.log | Koji-task-ID]
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ xmvn-builddep build.log

gzip: stdin: unexpected end of file
Traceback (most recent call last):
  File "/usr/share/java-utils/builddep.py", line 63, in <module>
    et = etree.parse(args[0]).getroot()
  File "src/lxml/etree.pyx", line 3536, in lxml.etree.parse
  File "src/lxml/parser.pxi", line 1875, in lxml.etree._parseDocument
  File "src/lxml/parser.pxi", line 1901, in lxml.etree._parseDocumentFromURL
  File "src/lxml/parser.pxi", line 1805, in lxml.etree._parseDocFromFile
  File "src/lxml/parser.pxi", line 1177, in lxml.etree._BaseParser._parseDocFromFile
  File "src/lxml/parser.pxi", line 615, in lxml.etree._ParserContext._handleParseResultDoc
  File "src/lxml/parser.pxi", line 725, in lxml.etree._handleParseResult
  File "src/lxml/parser.pxi", line 654, in lxml.etree._raiseParseError
  File "/dev/stdin", line 1
lxml.etree.XMLSyntaxError: Document is empty, line 1, column 1
BuildRequires:  maven-local
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End of the test.'
+ message='End of the test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of the test.'
Wed Feb 15 01:45:44 2023 - INFO  - End of the test.
+ CASE_RESULT 0
+ case_re=0
+ test -z ''
+ test 0 -eq 0
+ LOG_INFO 'succeed to execute the case.'
+ message='succeed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'succeed to execute the case.'
Wed Feb 15 01:45:46 2023 - INFO  - succeed to execute the case.
+ exec_result=
+ exit 0
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Wed Feb 15 01:45:49 2023 - INFO  - Start to restore the test environment.
+ rm -rf build.log .gradle build
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Wed Feb 15 01:45:51 2023 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'Finish restoring the test environment.'
+ message='Finish restoring the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish restoring the test environment.'
Wed Feb 15 01:45:54 2023 - INFO  - Finish restoring the test environment.
